src/index.ts

```diff
@@ -0,0 +1,12 @@
+import { ApolloServer } from "@apollo/server"
+import { startStandaloneServer } from "@apollo/server/standalone"
+import { resolvers } from "./resolver"
+import { typeDefs } from "./schema"
+
+const pureServer = new ApolloServer({ typeDefs, resolvers })
+
+;(async function () {
+  return (await startStandaloneServer(pureServer)).url
+})()
```

Comment:
nao precisa desse url ne?
poderia ser apenas

```suggestion
;(async function () {
  await startStandaloneServer(pureServer);
})()
```

Reply:
É verdade, esse url seria pego no retorno e jogado como o endereço de onde está rodando. 
Porém eu nao conseguia acessar fora, nem de outras formas que tentei. E quando pega é um objeto.